upi/metal/main.tf

```diff
@@ -100,27 +100,29 @@ locals {
 }
```

Comment:
Remove the local `packet_facility` variable, as it is unused (and has been unused).

Reply:
Fixed.